pyEOS/eos.py

```diff
@@ -157,17 +157,29 @@ def load_candidate_config(self, filename=None, config=None):
         else:
             self.candidate_config.load_config(config=config)
 
-    def compare_config(self):
+    def compare_replace_config(self):
```

Comment:
I wouldn´t change the name to be honest. In the long, run, when EOS support the config session there will be a kind of "show | compare" as with JunOS.